Probe/fcmHandler.go

```diff
@@ -0,0 +1,92 @@
+/*
+ *  Copyright 2020 Google LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package main
+
+import (
+	"errors"
+	"os/exec"
+	"regexp"
+	"strings"
+	"time"
+)
+
+var fcmAuthToken string
+var deadline = time.Now()
```

Comment:
Not quite sure why you have these global vars?

Reply:
addressed in 1e6bbce